src/server/api/controllers/reviews.controller.js

```diff
@@ -16,14 +16,57 @@ const getReviewsByUserId = async (id) => {
   try {
```

Comment:
@KavyaReddy732 Please comment out "that part that we get reviews for specific cv"  from ```reviews.router.js``` (line 23-28) here for now. To get right reviews from api/reveiws/{userid} correctly. Then we don't have to add extra files to Profile PR which's already supper crowded. good job :+1:

Reply:
@pdsorensen As Kavya explained, we need to comment out the first get params from reviews.router to take info from api/reviews/{id}  in the way ```SentReviewsComponent``` needed,, so we need to comment them here, not into the profile pr which is very crowded now.